packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -147,7 +147,7 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
     return (
       <div className={ textFieldClassName }>
         { label && <Label htmlFor={ this._id }>{ label }</Label> }
-        <div className={ css(styles.fieldGroup) }>
+        <div className={ css(styles.fieldGroup, isFocused && styles.fieldGroupIsFocused) }>
```

Comment:
I would suggest seperating out separate lines for each rule, but yes, I'd like to move away from the object notation, for example:

```tsx
<div 
  className={ 
    css(
      styles.fieldGroup,
      isFocused && styles.fieldGroupIsFocused
    ) }
/>
```
FWIW: when we introduce glamor, we need to normalize what the `css` helper does. In glamor, it can take objects that hold styles:

```ts
css({
   background: red
 })
```

whereas ours takes strings. I'd love a helper that can take both:

```ts
css(
  'myStringClassName',
  { background: 'red' }
)
```

Which should return a string `myStringClassName css-123abc`.